docs/components/forms.rst

```diff
@@ -268,6 +268,52 @@ To change the order of fields on your Form:
 #. Click the field you would like to move
 #. Drag the field to a new location
 
+.. vale off
+
+Progressive profiling
+*********************
+.. vale on
+
+Progressive profiling is a powerful feature used to reduce the length of Forms by hiding all the fields that are already known. This prevents your Contacts from feeling overwhelmed by massive Forms and even reduces the time it takes to fill out a Form if fields are already known to your Mautic instance and thus hidden for the Contact.
+
+Configuration
+==============
+
+There are two ways to configure a Form Field to only display when the asked values are unknown.
```

Comment:
```suggestion
There are two ways to configure a Form field to only display when the asked values are unknown.
```

I _think_ only Form should be capitalised here.

Reply:
Yes but it was giving a warning earlier so i capitalized the fields as well.